web/app/themes/mitlib-parent/functions.php

```diff
@@ -494,6 +494,17 @@ function customize_body_class( $classes ) {
 }
 add_filter( 'body_class', 'Mitlib\Parent\customize_body_class' );
 
+/**
+ * Defines a new value for the length of the excerpt.
+ *
+ * @param int $length Number of words to include in the excerpt.
+ * @return int The maximum number of words to include from the excerpt.
+ */
+function custom_excerpt_length( $length ) {
+	return 9999;
```

Comment:
This size was chosen as a value that should be big enough for no pages to be truncated. Graham and I have looked at the result of a crawl with this in place (in TIMX-253 ), and found that the longest page we have is currently 9,066 words (second longest is ~5,600 words).

Without this code change, the description field would be truncated at 55 words - so there are two decisions being made here:
1. We should change the description length away from its present value of 55 words, and
2. The description length should be large enough that no page is truncated.

I definitely think that we should not be using 55 words for this value. In terms of what the proper value is, I'm happy to discuss whether we should be imposing some sort of length limit. However, I think the preferrable setup at that point would be to actually provide a summary in the description field, rather than a truncated copy of the content.

Reply:
Gotcha. I have to sit with this a bit more. I definitely agree 55 is not the correct number so "bigger than 55" I'm on board with. I'll likely want to chat synchronously on this next week to better understand our intent and options. Thanks for the extra details!

Reply:
@matt-bernhardt which specific opengraph tag are we populating that is affected by this change?

Reply:
The affected tag is `og:description` - there's a link in https://mitlibraries.atlassian.net/browse/TIMX-253 to the most-affected page ( "Signers of the 1861 act of association" on the MIT History site)